src/frontend/js/components/CourseProductCourseRuns/CourseRunList.tsx

```diff
@@ -0,0 +1,75 @@
+import { Children } from 'react';
```

Comment:
I can't find the test file for this component ? each component shouldn't be tested independently? please can you add a file by component

Reply:
ok I found it, you've centralized all the tests in another file